libcloudforensics/providers/gcp/internal/storage.py

```diff
@@ -239,3 +237,38 @@ def GetBucketSize(self,
             elif val > ret[bucket]:
               ret[bucket] = val
     return ret
+
+  def CreateBucket(
+      self,
+      bucket: str,
+      labels: Optional[Dict[str, str]] = None,
+      predefinedacl: Optional[str] = 'private',
```

Comment:
See comment above for the use of `Optional`
Same below

Reply:
Done